### ISSUE TITLE
DOS_ 02_ Town05.xml 

### ISSUE THREAD

**Issue by a1wj1** [opendilab/DOS#7]
Hello, I am using DOS_ 02_ Town05.xml ran a route experiment and I couldn't see any occlusion scenes in the window. My agent called auto_ Pilot. py.  May I ask why? In addition, the car has been parked on this interface for a long time, and Carla has not been stuck. I don't know why？

![1689218243045](https://github.com/opendilab/DOS/assets/38905689/7eb4f3a2-4622-4b0b-8697-94d320121c43)

**Comment by deepcs233** [comment]
Hi!
Did you set DOS_ 02_ Town05.json correctly? And which route id is this?

By the way, i recommend you use a agent that do not cheat like interfuser agent, because auto_ Pilot. py can obtain the information of all objects although some of them are occluded.

**Comment by a1wj1** [comment]
![1689230976772](https://github.com/opendilab/DOS/assets/38905689/b8e647b0-299f-41a0-91b8-0f05e4513f3c)
I found that pedestrians wait for the first two yellow vans to pass before starting to cross the road. Is this not consistent with the description in the paper?

**Comment by a1wj1** [comment]
I just want to see the scene display of obstructed routes, but I haven't considered testing the model yet

**Comment by a1wj1** [comment]
![1689232323439](https://github.com/opendilab/DOS/assets/38905689/47582e01-f0f5-4450-9647-941aa51ab400)
In this case, the pedestrians did not move, shouldn't they have crossed the road？

**Comment by deepcs233** [comment]
Hi!
Please provide your run scirpt, and the cheated agent like 'auto_agent' will wait the occluded pedestrians. So i recommend you to change the agent. The 'occlusion'  of pedestrians denotes the view from the driver, not the birdview or the view with a certain height like the image you provided.


> I found that pedestrians wait for the first two yellow vans to pass before starting to cross the road. Is this not consistent with the description in the paper?

It's hard to develop `pedestrians suddenly emerge from the sidewalks` in the existing carla framework, so we initlize two pedestrians at the sidewalks to simulate this scenario. But from the driver view, they are the same.

> In this case, the pedestrians did not move, shouldn't they have crossed the road？

The pedestrians will cross the road when the ego car  is close enough to them. But your cheated agent will detect them and wait them alghough they are occluded. This can get into a deadlock-like state. So i recommend you to change a normal agent without using extra information.

**Comment by a1wj1** [comment]
I changed the agent but the same effect was achieved. I used the agent from the TCP code(https://github.com/OpenDriveLab/TCP/). Perhaps this agent also stopped early.

**Comment by a1wj1** [comment]
I understand that pedestrians only begin to cross the road when the autonomous vehicle approaches the intersection. If the autonomous vehicle stops at a distance, pedestrians will not move, while the tow yellow vans is only affected by traffic lights. May I ask if this understanding is correct?

**Comment by a1wj1** [comment]
![1689234823145](https://github.com/opendilab/DOS/assets/38905689/4668f82b-714a-4b6b-9b6d-66a17f3ac602)
In this situation, the autonomous vehicle is already very close to the pedestrian, but the pedestrian has not moved either? Is this also a deadlock-like state？

**Comment by deepcs233** [comment]
Could you provide the run script and the image from the driver view？
If the car doesn't see the pedestrian but stops early, we do not consider this type of scenarios. Could you try other routes?

> the tow yellow vans is only affected by traffic lights

Maybe it was stopped for too long? We didn't find this situation in our tests, these trucks would remain stationary until the end of the scenario.

**Comment by a1wj1** [comment]
OK，I will conduct more tests, and if there are any further questions, I will ask you again. Thank you.
